### PR TITLE
Create finetune.py

### PR DESCRIPTION
**Pull request by hassaanQadir**
This is script 1 out of 2 accompanying my 30Aug24 Fine-Tuning Llama3 article.

### CODE REVIEW

File: finetuning/llama/finetune.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+@function(
+    volumes=[Volume(name="llama-ft", mount_path=MOUNT_PATH)],
+    image=Image(
+        python_packages=["transformers", "torch", "datasets", "peft", "bitsandbytes"]
+    ),
+    gpu="A100-40",
+    cpu=4,
+)
+# finetune.py
```
Comment (mernit) [inline finetuning/llama/finetune.py:18]: remote comment

File: finetuning/llama/finetune.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
```
Comment (mernit) [inline finetuning/llama/finetune.py:1]: add comment at the top of the file that says how to run this script on beam

**Review by mernit — CHANGES_REQUESTED** [review]

File: finetuning/llama/finetune.py (hunk 2 above)
Comment (hassaanQadir) [reply finetuning/llama/finetune.py:1]: I added 
"Deploy to beam by running `$ python finetune.py` in the terminal"

**Review by hassaanQadir — COMMENTED** [review]
I added "Deploy to beam by running `$ python finetune.py` in the terminal"

**Review by hassaanQadir — COMMENTED** [review]
I fixed the beam deploy command to be
`$ beam deploy finetune.py:llama_fine_tune`

**Pull request merged**